bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/FileAssetIngestor.java

```diff
@@ -62,11 +63,35 @@ public FileAssetIngestor(MimeTypeService mimeTypeService) {
 
     @FormField(
             name = "Source",
-            description = "Source folder for content ingestion which can be a local folder or SFTP url with user/password",
-            hint = "/var/mycontent, /mnt/all_the_things, sftp://user:password@host[:port]/base/path...",
+            description = "Source folder for content ingestion which can be a local folder or SFTP url",
+            hint = "/var/mycontent, /mnt/all_the_things, sftp://host[:port]/base/path...",
             required = true
     )
     String fileBasePath;
+
+    @FormField(
+            name = "Connection timeout",
+            description = "HTTP Connection timeout (in milliseconds) for SFTP connection",
```

Comment:
```suggestion
            description = "Connection timeout (in milliseconds) for SFTP connection",
```

:)

Reply:
oops :), fixed in https://github.com/cylinder-y/acs-aem-commons/commit/d643945fb2b333d4886fa57252431090f897c671